npm/test/db/db.test.ts

```diff
@@ -206,6 +208,9 @@ tap.test('dbs', async () => {
         }
       );
 
+      // Prevent adding records with same timestamp
+      await new Promise((resolve) => setTimeout(resolve, 3000));
```

Comment:
What is this for? Does it add a lot of total delay to the CI/CD step?

Reply:
@deepakprabhakara We have tests that check the order of the records (asc or desc). I noticed records are randomly added with the same timestamp, and the records' order is not guaranteed while fetching, so tests were failing. 
Maybe we can reduce the delay to 1 second, or I can remove the delay if you think it's not needed.